packages/lex-cli/src/codegen/client.ts

```diff
@@ -146,6 +146,23 @@ const indexTs = (project: Project, schemas: Schema[], nsidTree: NsidNS[]) =>
         ].join('\n'),
       )
 
+    //= setHeader(key: string, value: string): void {
+    //=   this.xrpc.setHeader(key, value)
+    //= }
+    const setHeaderMethod = serviceClientCls.addMethod({
```

Comment:
new method. make sure i didn't do anything silly 😛

Reply:
Nailed it